packages/replay/src/integration.ts

```diff
@@ -145,15 +145,8 @@ export class Replay implements Integration {
       // collect fonts, but be aware that `sentry.io` needs to be an allowed
       // origin for playback
       collectFonts: true,
-      errorHandler: (err: Error) => {
-        try {
-          // @ts-expect-error Set this so that replay SDK can ignore errors originating from rrweb
-          err.__rrweb__ = true;
-        } catch {
-          // avoid any potential hazards here
-        }
-        // return true to suppress throwing the error inside of rrweb
-        return true;
+      errorHandler: (err: Error & {__rrweb__?: boolean}) => {
+        err.__rrweb__ = true;
```

Comment:
Hmmm so one problem here is that we _may_ throw/error on trying to set this on something that is e.g. frozen or similar. Can we instead still try-catch here and just re-throw certain kinds of exceptions, maybe?

I mean, _maybe_ this is actually fine, but we _could_ be getting other errors because of this, at some point, too 🤔

Reply:
Or actually, can we just do:

```js
errorHandler: (err: Error) => {
  try {
    // @ts-expect-error Set this so that replay SDK can ignore errors originating from rrweb
    err.__rrweb__ = true;
  } catch {
    // avoid any potential hazards here
  }
```

So still try-catch setting this prop, but bubble the error up to rrweb 🤔

Reply:
I updated this like this, so we can merge & release this!